cmd/temporalite/main.go

```diff
@@ -262,6 +269,21 @@ func buildCLI() *cli.App {
 				}
 				opts = append(opts, temporalite.WithLogger(logger))
 
+				configVals, err := getDynamicConfigValues(c.StringSlice(dynamicConfigValueFlag))
+				if err != nil {
+					return err
+				}
+				// If there is no config value for search attribute cache disabling,
+				// default it to true
+				if len(configVals[dynamicconfig.ForceSearchAttributesCacheRefreshOnRead]) == 0 {
+					configVals[dynamicconfig.ForceSearchAttributesCacheRefreshOnRead] = []dynamicconfig.ConstrainedValue{
+						{Value: true},
+					}
+				}
```

Comment:
Also, instead of setting a default here, I think we should probably do it inside `temporalite.NewServer` so that library users get the same behavior.

Reply:
I originally had the same concerns but thought Temporalite was going for "development defaults" even in the CLI. I would be happy only setting this in `temporaltest`. Let me confirm with @bergundy who wanted this defauted in the CLI.

Reply:
This makes me think that having the `temporalite "yolo"` command for common test setup (includes `--ephemeral` too) is starting to make sense.

Reply:
IMO Temporalite values go something like this

1. User experience (for development use cases)
2. Security
3. Performance
4. Simplicity
5. Configurability

So I'm very happy to trade off some performance for a better development experience. We can always add a `--production` flag to swap around a few defaults and keep things simple.

Another thought is that we could add flags to register search attributes at startup like we do for namespaces, and essentially have the best of both worlds since we'd be able to ensure that each of those SAs is registered synchronously at startup time, without disabling caching completely.

Reply:
> Another thought is that we could add flags to register search attributes at startup like we do for namespaces. This would essentially be the best of both worlds since we'd be able to ensure that each of those SAs is registered synchronously, without disabling caching completely.

I'd rather users do this with the SDK so their tests can run against any server implementation without customization.

Reply:
Removed default from CLI